tests/manage/pv_services/test_expansion_snapshot_clone.py

```diff
@@ -2,22 +2,43 @@
 import pytest
 
 from ocs_ci.helpers import helpers
+from ocs_ci.helpers.helpers import default_storage_class
 from ocs_ci.ocs import constants
 from ocs_ci.ocs.resources import pod
 from ocs_ci.framework.testlib import (
     ManageTest,
     tier2,
     skipif_ocs_version,
     skipif_ocp_version,
+    polarion_id,
 )
 
 log = logging.getLogger(__name__)
 
 
 @tier2
-@skipif_ocs_version("<4.6")
 @skipif_ocp_version("<4.6")
-@pytest.mark.polarion_id("OCS-2408")
+@pytest.mark.parametrize(
+    argnames=["pvc_create_sc_type", "restore_sc_type"],
+    argvalues=[
+        pytest.param(
+            *["thin", "thin"],
```

Comment:
The words "thin", "thick" I would replace with constants

Reply:
These parameters are used to distinguish between the usage of thick and thin provision storage class in this test case. I don't think the value of these parameters can be made as a standard.